python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -30,11 +30,11 @@ class OllamaChatCompletion(TextCompletionClientBase, ChatCompletionClientBase, A
     Make sure to have the ollama service running either locally or remotely.
 
     Arguments:
-        ai_model_id {str} -- Ollama model name, see https://ollama.ai/library
+        ai_model_id {str} -- Ollama model name, see https://ollama.ai/library, defaults to llama2
         url {Optional[Union[str, HttpUrl]]} -- URL of the Ollama server, defaults to http://localhost:11434/api/chat
         session {Optional[aiohttp.ClientSession]} -- Optional client session to use for requests.
     """
-
+    ai_model_id: str = "llama2"
```

Comment:
This field is already part of AI client base, and has important annotation. Should be overwritten if necessary with those.

Reply:
I see what @eavanvalkenburg is pointing out. You'll want to follow how it's done in another example, like [OpenAI chat completion](https://github.com/microsoft/semantic-kernel/blob/e4874bb79d7c4f50c6975353ca9032f2982d4a60/python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py#L177C9-L178C52).

Reply:
I am seeing in this example that before the ollama call, the `request_settings.ai_model_id = ''` which means it's just empty and not None.

Reply:
new commit added, I use same code style with [OpenAI chat completion](https://github.com/microsoft/semantic-kernel/blob/e4874bb79d7c4f50c6975353ca9032f2982d4a60/python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py#L177C9-L178C52).